torch/distributed/pipelining/schedules.py

```diff
@@ -565,6 +565,7 @@ def __init__(
         args_chunk_spec: Optional[Tuple[TensorChunkSpec, ...]] = None,
         kwargs_chunk_spec: Optional[Dict[str, TensorChunkSpec]] = None,
         output_merge_spec: Optional[Union[Dict[str, Any], Tuple[Any]]] = None,
+        stage_index_to_group_rank: Optional[Dict[int, int]] = None,
```

Comment:
Perhaps we should create 2 classes for 'PipelineScheduleLooped' and 'PipelineScheduleV' that provide the right default mapping. It wouldn't preclude a user creating a new type of mapping later but it would be a convenience and cover the known cases

Reply:
I think I would be okay with this, so then PipelineScheduleLooped always ensures that `rank = stage_index % group_size` and we don't allow the `stage_index_to_group_rank` arg?

I believe it should be a name other than`PipelineScheduleV` since the `stage_index_to_group_rank` provides any arbitrary stage ordering, not necessarily just a "V-shape"